src/widgets/_views.scss

```diff
@@ -17,4 +17,17 @@ scrolledwindow {
 .view {
     background-color: bg-color(1);
     color: $fg-color;
+
+    &.terminal {
+        background-color: #252e32;
+        color: #94a3a5;
+        font-family: monospace;
+
+        text {
+            selection {
```

Comment:
can probably either remove `text` or combine this to be `text selection {}`

Reply:
Combined they work. Fixed.